src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php

```diff
@@ -166,15 +168,13 @@ private function getPropertyMetadata(ResourceMetadata $resourceMetadata, string
 
         foreach ($this->propertyNameCollectionFactory->create($resourceClass, $options) as $propertyName) {
```

Comment:
I may think that this is bugging, and that we are not calling the good property with the good groups, @api-platform/core-team  any opinion ?

If you run this with the Dummy entity, you will see that this is not totally wrong